documentation/pom.xml

```diff
@@ -1,23 +1,23 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-<parent>
-    <groupId>com.devonfw.tools</groupId>
-    <artifactId>devonfw-docgen-pdf</artifactId>
-    <version>5.0.1</version>
-</parent>
-    <groupId>com.devonfw.tools.sonar-devon4j-plugin</groupId>
-    <artifactId>devonfw-sonar-devon4j-plugin-doc</artifactId>
-    <version>${revision}${changelist}</version>
-    <packaging>pom</packaging>
-    <name>${project.artifactId}</name>
-    <description>Documentation for devonfw-sonar-devon4j-plugin</description>
+    <parent>
+        <groupId>com.devonfw</groupId>
+        <artifactId>maven-parent</artifactId>
+        <version>5</version>
+    </parent>
 
+    <groupId>com.devonfw.tools.sonar-devon4j-plugin</groupId>
+    <artifactId>devonfw-docgen-pdff</artifactId>
```

Comment:
Looks like a typo and this does make sense to me. Why dont we still inherit this from `devonfw-docgen-pdf`?